_sections/30-bindings.md

```diff
@@ -112,8 +112,8 @@ A subset of [XPath 1.0 functions](http://www.w3.org/TR/xpath/#corelib), some fun
 <a id="fn:boolean-from-string" href="#fn:boolean-from-string">`boolean-from-string(string arg)`</a> | Returns true if arg is "true" or "1", otherwise returns false.
 <a id="fn:not" href="#fn:not">`not(boolean arg)`</a> | As in [XPath 1.0](http://www.w3.org/TR/xpath/#function-not).
 <a id="fn:number" href="#fn:number">`number(* arg)`</a> | As in [XPath 1.0](http://www.w3.org/TR/xpath/#function-number).
-<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | Converts dateTime value to a number.
-<a id="fn:decimal-time" href="#fn:decimal-time">`decimal-time(time value)`</a> | Converts time value to a number.
+<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | Converts dateTime value to a number in the format used by Excel. The number before the decimal marker is the number of days since January 1, 1970. The number after the decimal marker represents the time as a fractional date.
```

Comment:
Wow. I would have never ever guessed this! The number after the decimal marker means 3:00am is x.25 and noon is x.50?

Reply:
We should probably add "UTC" after 1970. I wonder if it would be easier to just leave out the after-decimal part. It seems unnecessary especially if we also take timezone into consideration.

Reply:
When you say leave it out, you mean not explicitly explain it because it's implied?